### ISSUE TITLE
Have only one package.json for this project

### ISSUE THREAD

**Issue by travs** [melonproject/satellite#14]
Right now there are two package.json files, one in `truffle/`, and another in `meteor/`.
This is bad for versioning and makes the installed repo much larger (two `node_modules` dirs).

Should either make two separate repos for this, or unify the `package.json` into the repo's root.

**Comment by travs** [comment]
One option is to use [Lerna](https://lernajs.io/) and the [monorepo architecture](https://github.com/babel/babel/blob/7.0/doc/design/monorepo.md).

[Experimental branch](https://github.com/melonproject/satellite/tree/lerna) is here, so you can see project structure with Lerna

**Comment by retotrinkler** [comment]
even though Lerna seems an elegant solution, I would have a slight preference towards two repositories; or to incorporate the satellite contract w/in the protocol repo.

then we can packages satellites also in a npm module and only publlish .json files for examples to make it as light as possible.

also bec i see satellite and portal app to be combined in one app reasonably soon - maybe even for 0.2.0 release of portal

**Comment by travs** [comment]
Yes I definitely agree, Lerna is a bit overkill for just two packages.
Make a separate repo for `satellite-app` or whatever name, even if it will be pushed into portal in the next release, so we can keep an accurate changelog and have better versioning for the Satellite contracts. What do you think?
